packages/schematics/angular/library/schema.json

```diff
@@ -43,6 +43,11 @@
       "type": "boolean",
       "default": false,
       "description": "Do not update \"tsconfig.json\" to add a path mapping for the new library. The path mapping is needed to use the library in an app, but can be disabled here to simplify development."
+    },
+    "directory": {
```

Comment:
Can we rename this to `projectRoot` similar like the option in the application schematic? For consistence in this case the newProjectRoot and folderName should be ignored.


https://github.com/angular/angular-cli/blob/6987b446557dadce32c938eff9b3f925e400af78/packages/schematics/angular/application/index.ts#L248-L261